### PR TITLE
Port changes made in release to develop

### PR DESCRIPTION
**Pull request by ximinez**
Includes 48396ae, 1edc5e5, and 75af4ed.

### CODE REVIEW

File: src/ripple/app/ledger/impl/LedgerConsensusImp.cpp [hunk 1]
```diff
@@ -1919,14 +1919,7 @@ void LedgerConsensusImp::addLoad(STValidation::ref val)
...
-    std::uint32_t loadBase = app_.getFeeTrack().getLoadBase();
-
-    // Hard code the minimum fee for now:
-    std::uint32_t const minFee = 10000;
-    std::uint32_t const refFee = 10;
-    fee = std::max(fee, (minFee * loadBase + refFee - 1) / refFee);
-
-    if (fee > loadBase)
+    if (fee > app_.getFeeTrack().getLoadBase())
         val->setFieldU32(sfLoadFee, fee);
```
Comment (wilsonianb) [inline src/ripple/app/ledger/impl/LedgerConsensusImp.cpp]: The `ripple.app.Subscribe` test used to get a validation that included `sfLoadFee`, but now it doesn't.
I'm guessing this caused the change. Is that expected?
https://github.com/ripple/rippled/blob/develop/src/ripple/rpc/tests/Subscribe.test.cpp#L316

Comment (ximinez) [inline src/ripple/app/ledger/impl/LedgerConsensusImp.cpp]: I think yes. Without the manual bump, the fee is now back to it's normal lower value, which is unlikely to be larger than the load base, particularly during unit tests.

Comment (wilsonianb) [inline src/ripple/app/ledger/impl/LedgerConsensusImp.cpp]: Cool, then removing this line should fix the failing test:
https://github.com/ripple/rippled/blob/develop/src/ripple/rpc/tests/Subscribe.test.cpp#L316

Comment (ximinez) [inline src/ripple/app/ledger/impl/LedgerConsensusImp.cpp]: Failing test? Ugh. I missed that. Yeah, it should probably fix. I'll do that ASAP.

**Comment by nbougalis** [comment]
:+1:

**Comment by nbougalis** [comment]
Merged as ceb01fb6a34b87bd91cdb4d693b11326cd726a14, 321e2a94fe8b9ba61baff3bcbc9852665b084606 and 7f52249e407e362e2c3bbcb4decfc62df0efe718.

**Pull request closed without merging**